### PR TITLE
Header only lib

### PR DESCRIPTION
**Pull request by costashatz**
No need to have a library for one class library...

### CODE REVIEW

File: hexapod_controller/include/hexapod_controller/hexapod_controller_simple.hpp [hunk 1]
```diff
@@ -18,18 +18,181 @@ namespace hexapod_controller {
...
+            _legs4commands.push_back(_control_signal(ctrl[24], ctrl[25], ctrl[26]));
+            _legs4commands.push_back(_control_signal(ctrl[27], ctrl[28], ctrl[29]));
+            _legs4commands.push_back(_control_signal(ctrl[27], ctrl[28], ctrl[29]));
+
+            _legs5commands.push_back(_control_signal(ctrl[30], ctrl[31], ctrl[32]));
+            _legs5commands.push_back(_control_signal(ctrl[33], ctrl[34], ctrl[35]));
+            _legs5commands.push_back(_control_signal(ctrl[33], ctrl[34], ctrl[35]));
+        }
+
+        std::vector<double> parameters()
```
Comment (dogoepp) [inline hexapod_controller/include/hexapod_controller/hexapod_controller_simple.hpp]: According to good practice, getters should return const references and should be declared const when possible. We might replace it with

``` C++
const std::vector<double> parameters() const
```

Same goes for `broken_legs()`.

Comment (costashatz) [inline hexapod_controller/include/hexapod_controller/hexapod_controller_simple.hpp]: Yeap you are right..Will do..

**Comment by costashatz** [comment]
@dogoepp this should ok now..

File: hexapod_controller/include/hexapod_controller/hexapod_controller_simple.hpp [hunk 2]
```diff
@@ -71,12 +71,12 @@ namespace hexapod_controller {
             _broken_legs = broken_legs;
         }
 
-        std::vector<int> broken_legs()
+        std::vector<int> broken_legs() const
```
Comment (dogoepp) [inline hexapod_controller/include/hexapod_controller/hexapod_controller_simple.hpp]: Why isn't it more like

``` C++
const std::vector<double>& parameters() const;
const std::vector<int>& broken_legs() const;
```

These method declarations give read-only access to member variables, copy-less.

Comment (costashatz) [inline hexapod_controller/include/hexapod_controller/hexapod_controller_simple.hpp]: You're right.. Will do..

**Comment by costashatz** [comment]
> Why isn't it more like
> 
> const std::vector<double>& parameters() const;
> const std::vector<int>& broken_legs() const;
> These method declarations give read-only access to member variables, copy-less.

Fixed...

**Comment by dogoepp** [comment]
If it compiles, **OK** for me.

**Comment by costashatz** [comment]
**Merging..**

**Pull request merged**